src/s2_fos/model.py

```diff
@@ -0,0 +1,55 @@
+"""
+Implement your model here! Or not.
+"""
+import os
+
+import torch
+import numpy as np
+from transformers import (
+    AutoTokenizer, AutoConfig, AutoModelForSequenceClassification,
+)
+
+
+class PredictProbabilities:
+
+    def __init__(self, model_path=None):
+        # Load the trained model and tokenizer
+        self.device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+        if model_path:
+            full_model_path = os.path.join(model_path, 'pytorch_model.bin')
+        else:
+            full_model_path = 'allenai/scibert_scivocab_uncased_fielf_of_study'
+
+        self.config = AutoConfig.from_pretrained(model_path)
```

Comment:
It makes more sense to just hardcode the model path in the constants file and put it here so the user doesn't need to download it themselves during the install process. After all, huggingface has great caching functionality etc.

Reply:
Not clear on suggestion, but I moved the model_name to the constants

Reply:
This is a relative path and there are relative paths all over. So if the user is not in the install directory then  it won't work. I recommend using a method so you can just import the absolute path of the data directly and use it .

See here https://github.com/allenai/S2AFF/blob/main/s2aff/consts.py